example.json

```diff
@@ -6,7 +6,8 @@
     "_button": {
       "text": "{{_globals._extensions._tutor.hideFeedback}}",
       "ariaLabel": "{{_globals._extensions._tutor.hideFeedback}}"
-    }
+    },
+    "altTitle": "Feedback"
```

Comment:
Just to reiterate, this is never going to be used once the question:showFeedback event passes an `altTitle` through. You may as well remove it.